Runtime/Game/LootLockerSDKManager.cs

```diff
@@ -463,36 +335,24 @@ public static void EndSession(string deviceId, Action<LootLockerSessionResponse>
         }
         #endregion
 
-        #region White Label Login
+        #region White Label
 
         /// <summary>
-        /// Create new user using the white label login system.
+        /// Log in a white label user with the given email and password combination, verify user, and start a session.
```

Comment:
Change "and start a session" to "and start a White Label session" to avoid confusion.